force-app/main/default/classes/SOQL_Test.cls

```diff
@@ -56,6 +56,50 @@ private class SOQL_Test {
         Assert.areEqual('SELECT Name, BillingCity FROM Account', soql);
     }
 
+    @IsTest
+    static void withTwoField() {
```

Comment:
Please use plural to align with other methods. It should be named `withTwoFields()`